exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/xsort/managed/TestSorter.java

```diff
@@ -59,7 +59,7 @@
 @Category(OperatorTest.class)
 public class TestSorter extends DrillTest {
 
-  public static OperatorFixture fixture;
+  public volatile static OperatorFixture fixture;
```

Comment:
I was running into an issue where the fixture variable was consistently null when the test ran. This seemed impossible and I hypothesized that JUnit is secretly using two threads, one to call the static initializers and then another to actually execute the test methods. So I made the variable volatile and the issue went away. I agree this is weird and it raises the question why other tests don't have the same issue. I'll remove the volatile for now, if the bizarre issue surfaces again, I will get more details and post them here.

Reply:
Strange... So, if having volatile does solve a problem, just leave it in, but maybe add a comment that includes you note above so that we know what's what in the future.